superset-frontend/src/explore/components/AdhocMetricOption.jsx

```diff
@@ -53,7 +53,7 @@ export default class AdhocMetricOption extends React.PureComponent {
     // once the overlay has been opened, the metric/filter will never be
     // considered new again.
     this.props.adhocMetric.isNew = false;
-    this.setState({ overlayShown: false });
+    this.setState({ overlayShown: true });
```

Comment:
Is this related to the glyph changes? is it intentional?

Reply:
That was a minor bug I uncovered, as I change the icons to `caret-(left|right)`